### PR TITLE
fix: docs versioning bug

### PR DESCRIPTION
**Pull request by NeethuES-intel**
## PR Checklist

- [X] Added label to the Pull Request for easier discoverability and search
- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**490**

## Test Instructions if applicable
1) push a new version `mike deploy 1.0` 
2) run `mike serve` (if error, then upgrade mkdocs-material to latest version)
3) view doc at `http://localhost:8000/1.0
4) Verify if code block (bash/json) is broken on pages -  dev-tools/documentation.html or troubleshooting.html

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -73,7 +73,6 @@ extra_css:
 extra_javascript:
   - 'https://unpkg.com/lunr/lunr.js'
 markdown_extensions:
-  - codehilite
   - admonition
   - pymdownx.details
   - pymdownx.highlight
```
Comment (jim-wang-intel) [inline mkdocs.yml:78]:
```suggestion
  - pymdownx.highlight:
       pygments_lang_class: true
```

Comment (NeethuES-intel) [reply mkdocs.yml:78]: error was because codehilite is deprecated, so the page was broken

Comment (jim-wang-intel) [reply mkdocs.yml:78]: it is nice to have

Comment (NeethuES-intel) [reply mkdocs.yml]: updated 👍

**Review by jim-wang-intel — APPROVED** [review]

**Pull request merged**